### PR TITLE
chore: bound version constraints in a renovate-friendly way

### PR DESCRIPTION
**Pull request by cpcloud**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -21,36 +21,36 @@ classifiers = [
...
+clickhouse-sqlalchemy = { version = ">=0.1.4,<0.2", optional = true }
 dask = { version = "^2021.2.0", optional = true, extras = [
   "array",
   "dataframe"
 ] }
 geoalchemy2 = { version = ">=0.6,<0.10", optional = true }
 geopandas = { version = ">=0.6,<0.11", optional = true }
-graphviz = { version = "^0.16", optional = true }
-hdfs = { version = "^2", optional = true, extras = ["kerberos"] }
+graphviz = { version = ">=0.16,<0.20", optional = true }
```
Comment (kszucs) [inline pyproject.toml:43]: `<0.17`?

Comment (cpcloud) [reply pyproject.toml:43]: nah, this is updating graphviz as well, from the #3157

**Review by kszucs — APPROVED** [review]

**Pull request merged**